### DISCUSSION TITLE
How to change the color of Squiggly?

### DISCUSSION THREAD

**Discussion by srmm2** [sumatrapdfreader/sumatrapdf#3360] [Q&A] [answered]
I tried writing 'SquigglyColor = xxx' in advanced options, but it didn't work. How can I change it? Thanks!

**Comment by GitHubRulesOK** [comment]
Hmm
Currently you would need to edit source and recompile as those are held in the source file https://github.com/sumatrapdfreader/sumatrapdf/blob/master/do/settings_def.go

**Reply by srmm2** [reply]
Thanks!

**Reply by GitHubRulesOK** [reply]
Note the new UPDATE below allows you pre-set colours

For existing annotations  
You can as a user change colours after the event in the annotation editor (but currently no way to do that in bulk fashion.)

![image](https://user-images.githubusercontent.com/12754913/226118840-f3c6ef30-22a0-4219-be96-8b8502a8c0fd.png)

**Comment by kjk** [comment, marked as answer]
I've added `SquigglyColor` and `StrikeOutColor` advanced options in latest pre-release https://www.sumatrapdfreader.org/prerelease (15349 or later)

**Reply by srmm2** [reply]
That's great. Thanks~
